examples/create-machine.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const Machine = require('..');
+
+var options = {
+  "driver": "virtualbox"
```

Comment:
Unquote the property and use single quotes for the value: `driver: 'virtualbox'`.